src/components/button/button.js

```diff
@@ -0,0 +1,24 @@
+import { createElement } from "../../utils/elements";
+import singleArrowSrc from "../../assets/arrow-right.svg";
+
+export const createActionElement = () => {
+  const actionButton = createElement("button", {
```

Comment:
Did you solve the problem already? looks like your button hasn't got a class named "btn" so it can't be manipulated 👍